WORKSPACE

```diff
@@ -63,3 +64,75 @@ http_archive(
 load("@rules_java//java:repositories.bzl", "rules_java_dependencies", "rules_java_toolchains")
 rules_java_dependencies()
 rules_java_toolchains()
+
+#Add support for Dagger
+DAGGER_TAG = "2.28.1"
+DAGGER_SHA = "9e69ab2f9a47e0f74e71fe49098bea908c528aa02fa0c5995334447b310d0cdd"
+http_archive(
+    name = "dagger",
+    strip_prefix = "dagger-dagger-%s" % DAGGER_TAG,
+    sha256 = DAGGER_SHA,
+    urls = ["https://github.com/google/dagger/archive/dagger-%s.zip" % DAGGER_TAG],
+)
+
+load("@dagger//:workspace_defs.bzl", "DAGGER_ARTIFACTS", "DAGGER_REPOSITORIES")
+
+#Add support for Robolectric: https://github.com/robolectric/robolectric-bazel
+http_archive(
+    name = "robolectric",
+    urls = ["https://github.com/oppia/robolectric-bazel/archive/4.x-oppia-exclusive-rc02.tar.gz"],
+    strip_prefix = "robolectric-bazel-4.x-oppia-exclusive-rc02",
+)
+load("@robolectric//bazel:robolectric.bzl", "robolectric_repositories")
+robolectric_repositories()
+
+#Add support for Firebase Crashlytics
+
+git_repository(
+    name = "tools_android",
+    commit = "00e6f4b7bdd75911e33c618a9bc57bab7a6e8930",
+    remote = "https://github.com/bazelbuild/tools_android"
+)
+
+load("@tools_android//tools/googleservices:defs.bzl", "google_services_workspace_dependencies")
+google_services_workspace_dependencies()
+
+
+load("@rules_jvm_external//:defs.bzl", "maven_install")
+#TODO: Remove unused dependencies once android_local_test errors are resolved
+maven_install(
+    artifacts = DAGGER_ARTIFACTS + [
+        "org.robolectric:robolectric:4.3",
+        "androidx.appcompat:appcompat:1.0.2",
+        "com.android.support:support-annotations:28.0.0",
+        "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.2",
+        "androidx.core:core-ktx:1.0.1",
+        "org.jetbrains.kotlinx:kotlinx-coroutines-test:1.2.2",
+        "junit:junit:4.12",
+        "com.google.truth:truth:0.43",
+        "com.github.bumptech.glide:glide:4.11.0",
+        "com.github.bumptech.glide:compiler:4.11.0",
+        "com.caverock:androidsvg-aar:1.4",
+        "com.crashlytics.sdk.android:crashlytics:2.9.8",
+        "io.fabric.sdk.android:fabric:1.4.7",
+        "com.google.gms:google-services:4.3.3",
+        "androidx.lifecycle:lifecycle-livedata-ktx:2.2.0-alpha03",
+        "com.google.firebase:firebase-analytics-ktx:17.4.2",
+        "org.mockito:mockito-core:2.19.0",
+        "androidx.test.ext:junit:1.1.1",
+        "org.jetbrains.kotlin:kotlin-stdlib-jdk7:jar:1.3.72",
+        "org.jetbrains.kotlin:kotlin-test-junit:1.3.72",
+        "org.robolectric:annotations:4.3",
+        "com.crashlytics.sdk.android:crashlytics:2.9.8", #Firebase
```

Comment:
Do we need this `#Firebase` comment? 
I think the dependency is itself self-understanding.